iphone/cli/commands/_build.js

```diff
@@ -6537,12 +6560,12 @@ iOSBuilder.prototype.invokeXcodeBuild = function invokeXcodeBuild(next) {
 		this.target === 'dist-appstore' || this.target === 'dist-adhoc' ? 'archive' : 'build',
 		'-target', this.tiapp.name,
 		'-configuration', this.xcodeTarget,
-		'-scheme', this.tiapp.name.replace(/[-\W]/g, '_'),
+		'-scheme', this.sanitizedAppName(),
 		'-derivedDataPath', path.join(this.buildDir, 'DerivedData'),
+		'-UseNewBuildSystem=' + this.useNewBuildSystem ? 'YES' : 'NO',
```

Comment:
You need parenthesis around `this.useNewBuildSystem ? 'YES' : 'NO'`. The `+` has higher precedence than `?` `:`. This will likely address the issue @sgtcoolguy was having.

Reply:
I think the issue was that it was passed as 2 array args ([`-UseNewBuildSystem`, YES/NO])` before.

Reply:
*Cough* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals? 🙂 
```js
`-UseNewBuildSystem=${x ? 'YES' : 'NO'}`
```

Reply:
Sigh. Yeah, template literals.